cryocare/__init__.py

```diff
@@ -72,21 +71,16 @@ def defineBinaries(cls, env):
 
         # Create the environment
         installationCmd += 'conda create -y -n %s -c conda-forge -c anaconda python=3.8 ' \
-                           'cudnn=7.6.5=cuda10.1_0 && ' % CRYOCARE_ENV_NAME
+                           'cudatoolkit=11.0 cudnn=8.0 && ' % CRYOCARE_ENV_NAME
+        # 'keras-gpu=2.3.1 ' \
 
         # Activate new the environment
         installationCmd += 'conda activate %s && ' % CRYOCARE_ENV_NAME
 
-        # Install non-conda required packages
-        installationCmd += 'pip install tensorflow-gpu==2.3.3 && '
-        installationCmd += 'pip install mrcfile && '
-        installationCmd += 'pip install csbdeep '
-        # I had the same issue and was able to fix this by setting h5py < 3.0.0.
-        # Looks like here was a 3.0 release of h5py recently where they changed how strings are stored/read.
-        # https://github.com/keras-team/keras/issues/14265
-
-        # Install cryoCARE
-        installationCmd += 'pip install %s==%s &&' % (CRYOCARE, CRYOCARE_DEFAULT_VERSION)
+        # Install cryoCARE and the rest of dependencies
+        installationCmd += 'virtualenv==20.3.1 && '
```

Comment:
this command does nothing :)

Reply:
true

Reply:
You're both right. Fixed! Sorry for the delay, @azazellochg, I missed the notification.

Reply:
No problem. I was just installing the plugin for the first time and skimmed through the init code